src/evo/specialtx.h

```diff
@@ -15,27 +15,23 @@
 #include <vector>
```

Comment:
```suggestion
#include <optional>
#include <vector>
```

at some point it may not be included in-directly